akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/MultiNodeClusterShardingSpecSpec.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import scala.concurrent.duration._
+
+import akka.actor._
+import akka.cluster.MemberStatus
+import akka.testkit._
+import com.typesafe.config.ConfigFactory
+
+abstract class MultiNodeClusterShardingSpecConfig(mode: String, rememberEntities: Boolean)
+    extends MultiNodeClusterShardingConfig(mode, rememberEntities) {
+
+  val first = role("first")
+  val second = role("second")
+  val third = role("third")
+  val fourth = role("fourth")
+
+  nodeConfig(third)(ConfigFactory.parseString(s"""
+    akka.cluster.sharding.distributed-data.durable.lmdb.dir = $targetDir/sharding-third
+    """))
+
+}
+
+object PersistentClusterShardingRememberEntitiesEnabledConfig
+    extends MultiNodeClusterShardingSpecConfig(ClusterShardingSettings.StateStoreModePersistence, true)
+
+class PersistentClusterShardingMultiNodeSpecMultiJvmNode1
+    extends MultiNodeClusterShardingSpecSpec(PersistentClusterShardingRememberEntitiesEnabledConfig)
+class PersistentClusterShardingMultiNodeSpecMultiJvmNode2
+    extends MultiNodeClusterShardingSpecSpec(PersistentClusterShardingRememberEntitiesEnabledConfig)
+class PersistentClusterShardingMultiNodeSpecMultiJvmNode3
+    extends MultiNodeClusterShardingSpecSpec(PersistentClusterShardingRememberEntitiesEnabledConfig)
+class PersistentClusterShardingMultiNodeSpecMultiJvmNode4
+    extends MultiNodeClusterShardingSpecSpec(PersistentClusterShardingRememberEntitiesEnabledConfig)
+
+object DDataClusterShardingRememberEntitiesEnabledConfig
+    extends MultiNodeClusterShardingSpecConfig(ClusterShardingSettings.StateStoreModeDData, true)
+
+class DDataClusterShardingMultiNodeSpecMultiJvmNode1
+    extends MultiNodeClusterShardingSpecSpec(DDataClusterShardingRememberEntitiesEnabledConfig)
+class DDataClusterShardingMultiNodeSpecMultiJvmNode2
+    extends MultiNodeClusterShardingSpecSpec(DDataClusterShardingRememberEntitiesEnabledConfig)
+class DDataClusterShardingMultiNodeSpecMultiJvmNode3
+    extends MultiNodeClusterShardingSpecSpec(DDataClusterShardingRememberEntitiesEnabledConfig)
+class DDataClusterShardingMultiNodeSpecMultiJvmNode4
+    extends MultiNodeClusterShardingSpecSpec(DDataClusterShardingRememberEntitiesEnabledConfig)
+
+abstract class MultiNodeClusterShardingSpecSpec(multiNodeConfig: MultiNodeClusterShardingSpecConfig)
```

Comment:
Testing the base test.

Reply:
I don't think it makes sense to keep a test testing the test base class when we have started to use it, but I guess this is only temporary to check that it actually works?

Reply:
Yes, I'll remove it in https://github.com/akka/akka/issues/27749 - noted.